### PR TITLE
Add initial php sdk

### PR DESCRIPTION
**Pull request by stvyang**

### CODE REVIEW

**Comment by stvyang** [comment]
This is the MVP implementation (not confirmed yet) to be published on Packagist:
- There will be adjustments based on user feedback (which will be the developer of Indodana merchant plugin)
- We will make an official release after the implementation is confirmed
- PHP docs and unit tests will be added upon official release

File: php/src/IndodanaRequest.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  {
+    $bearerToken = IndodanaSecurity::generateBearerToken(
+      $apiKey,
+      $apiSecret
+    );
+
+    return [
+      'Content-type: application/json',
+      'Accept: application/json',
+      "Authorization: Bearer $bearerToken"
```
Comment (sendyhalim) [inline php/src/IndodanaRequest.php]: Minor use `"{$bearerToken}"`

File: php/src/IndodanaSecurity.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      $apiKey,
+      $nonce
+    );
+
+    $signature = self::generateSignature(
+      $content,
+      $apiSecret
+    );
+
+    return "$content:$signature";
```
Comment (sendyhalim) [inline php/src/IndodanaSecurity.php]: same for this

File: php/src/IndodanaSecurity.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  }
+
+  public static function getContent($apiKey, $nonce)
+  {
+    return "$apiKey:$nonce";
```
Comment (sendyhalim) [inline php/src/IndodanaSecurity.php]: same as above

**Pull request merged**